src/phosphor-rbmc-data-sync-mgr_main.cpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include <phosphor-logging/lg2.hpp>
+#include <sdbusplus/async/context.hpp>
+#include <sdbusplus/server/manager.hpp>
+#include <xyz/openbmc_project/DataSync/BMCData/common.hpp>
+
+int main()
+{
+    try
+    {
+        using BMCDataSync =
+            sdbusplus::common::xyz::openbmc_project::data_sync::BMCData;
+
+        sdbusplus::async::context ctx;
+        sdbusplus::server::manager_t objManager{ctx,
+                                                BMCDataSync::namespace_path};
+
+        // clang-tidy currently mangles this into something unreadable
```

Comment:
Since you don't have a .clang-tidy file to enable clang-tidy, I don't think you'd need to worry about the NOLINT stuff.

Though since you're starting from scratch, it may be a good time to add one, and just disable the checks that cause these complaints so the code doesn't have to worry about it.

(I was hoping they would alll be fixed in clang-19, but they weren't)

Reply:
Ah, I see. I misunderstood —I thought it was for Clang-Format and didn’t realize Clang-Tidy is different. Thanks for pointing that out! I’ll add clang tidy configuration.